src/software/simulation/physics/physics_ball.cpp

```diff
@@ -1,13 +1,15 @@
 #include "software/simulation/physics/physics_ball.h"
 
 #include "shared/constants.h"
+#include "software/new_geom/util/distance.h"
 #include "software/simulation/physics/box2d_util.h"
 #include "software/simulation/physics/physics_object_user_data.h"
 
 PhysicsBall::PhysicsBall(std::shared_ptr<b2World> world, const BallState &ball_state,
                          const double mass_kg, double restitution, double linear_damping)
     : in_flight_origin(std::nullopt),
       in_flight_distance_meters(0.0),
+      flight_angle_of_departure(Angle::zero()),
```

Comment:
Wondering why this is the default value and if it gets reset

Reply:
Thought zero was a reasonable default since the ball isn't in flight on creation, and yes it gets set by the `setInFlightForDistance` function

Reply:
I was wondering actually if it gets reset to 0 every time a chip lands

Reply:
It doesn't get reset, since it's only used when the ball is in flight.